File: CS50x/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        // Prompts for user input
        height = get_int("Height: ");
        // Check the condition height be in between 1 to 8 (both inclusive)
    }
    while (height < 1 || height > 8);

    // Iteration i for rows
    for (int i = 1; i <= height; i++)
    {
        // Iteration j for columns
        for (int j = height; j > i; j--)
        {
            printf(" ");
        }
        // Iteration k for hashes
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: CS50x/sentimental-mario-less/mario.py
# Promts user for pyramid height
from cs50 import get_int

height = get_int("Height: ")
while(height < 1 or height > 8):
    height = int(input("Height: "))

# Iteration i for rows
for i in range(1, height + 1):
    # Iteration j for columns
    for j in range(height, i, -1):
        print(" ", end='')
    # Iteration k for hashes
    for k in range(i):
        print("#", end='')
    print()
